### DISCUSSION TITLE
How can I deleted field name?

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9215] [Q&A] [answered]
I want when user chose percent  shown percent input field when chose amount fixed price shown while submiting form only chose field should be there either fixed amount or percent 
like this
"locations": [
                {
                    "location": "Main Boulevard",
                    "is_fixed": "1",         //Amount selected percent absent
                    "price": "100000"
                },
                {
                    "location": "Corner",
                    "is_fixed": "0",  // Percent selected price is absent
                    "percent": "5"
                }
My code sandbox
https://codesandbox.io/s/react-hook-form-forked-rdesqv?file=/src/components/forms/basics/DynamicFields.jsx

@bluebill1049 
@leapful 
@evocateur 
@aaronjensen

**Comment by bluebill1049** [comment, marked as answer]
> I want when user chose percent shown percent input field when chose amount fixed price shown while submiting form only chose field should be there either fixed amount or percent

you probably want to pick a price based on user selection, don't think you need to delete fields.
